### ISSUE TITLE
Checker should not complain about missing ext-json when PHP 8 is used

### ISSUE THREAD

**Issue by vudaltsov** [maglnet/ComposerRequireChecker#245]
See https://php.watch/versions/8.0/ext-json

**Comment by dkreuer** [comment]
Looking into it.

**Comment by Ocramius** [comment]
JSON is still part of an **extension** of PHP.

I'd say that PHP-SRC should stop reporting it as an extension-declared function, and instead move it to core directly? 

This should be empty ( https://3v4l.org/iShcS ):

```php
<?php

var_dump(get_extension_funcs('json'));
```

**Comment by dkreuer** [comment]
I agree, but won't try to change that in upstream :D .. and to be correct it would have to be empty for ^8.0 - which won't be possible due to already released versions. So for compatibility this case must be supported anyways, right?

**Comment by Ocramius** [comment]
TBH, just declare `ext-json` and be done with it :shrug:

**Comment by dkreuer** [comment]
I'm fine with that.

For documentation purposes:
> If one has `composer-require-checker` complaining about missing symbols from `ext-json`, please require `ext-json` in `composer.json`.

**Comment by Ocramius** [comment]
Closing here meanwhile :+1:
